modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/platform.bndrun

```diff
@@ -15,7 +15,6 @@ $h1 Check GETTING_STARTED.markdown#platform.bndrun for more information.
 	osgi.service;objectClass:List<String>='com.liferay.portal.kernel.portlet.PortletPreferencesFactory',\
 	osgi.service;objectClass:List<String>='com.liferay.portal.kernel.portlet.PortletURLFactory',\
 	osgi.service;objectClass:List<String>='com.liferay.portal.kernel.service.persistence.LayoutSetPersistence',\
-	osgi.service;objectClass:List<String>='com.liferay.portal.kernel.util.File',\
```

Comment:
@shuyangzhou I am not very sure about this change. 
Do you know what this does?